app/ConfCrypt/CLI/Engine.hs

```diff
@@ -92,14 +95,23 @@ run parsedArguments = do
         injectPrivateKey :: PrivateKey -> (ConfCryptFile, ()) -> (ConfCryptFile, TextKey PrivateKey)
         injectPrivateKey key (conf, _) = (conf, TextKey key)
 
+        hasToBeSaved (AC _ _) = True
```

Comment:
This is a nice addition, but it changes the semantics of the program quite a bit and belongs in a separate PR. In order to integreate with other tools more easily, `confcrypt` actually prints its output to stdout rather than performing inplace updates of the config file.

That being said, it would be  great to separately add this function with an `inplace` command line flag to let users select this behavior.

Reply:
OK. I have removed it for now and kept it for another PR.